### ISSUE TITLE
fixme: Include the multi-line comments in the message emitted

### ISSUE THREAD

**Issue by moylop260** [PyCQA/pylint#1028]
### Steps to reproduce
1. Create a file with the following code:
```python
# TODO: This is a comment very
#     large
```
2. Run the command: `pylint -d all -e fixme my_file.py`
3. See output: `W:  1, 0: TODO: This is a comment very (fixme)`

### Current behaviour
The message emitted just show the first line of the comment.

### Expected behaviour
Show the full `TODO` comment
See a output similar to `W:  1, 0: TODO: This is a comment very large (fixme)`
This is useful when you have a `line-too-long` configured and a multiline comment is required.

### pylint --version output
```bash
pylint 2.0.0,
astroid 1.5.0
Python 2.7.10 (default, Oct 23 2015, 19:19:21)
[GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.0.59.5)]
```

NOTE: I could work on this If looks good to you

**Comment by PCManticore** [comment]
What if the comment is too long? I did saw TODO comments that were spanning on more than 4 lines, I wouldn't expect these to be merged into one line, displayed by Pylint.

I think the behavior we have right now is enough for most of the cases, as long as it shows the originating line, that would be enough.

**Comment by moylop260** [comment]
Thanks for feedback.
Ok I got it.

I'm working in a `rst` summary report of `To Do` of a python project based on comments, but the info displayed It was useless like as `This is a comment very.` ("very" what?)

If we merge them into one line we will have a useful content.

But you are right, the output could be very large for this case.
